### DISCUSSION TITLE
Are command-running backticks really useful?

### DISCUSSION THREAD

**Discussion by ineu** [nushell/nushell#10056] [General]
There's an issue https://github.com/nushell/nushell/issues/9995 caused by the fact the backticks can be used in two meanings:
1. A string delimiter
2. A way to run external commands.

I wonder if 2 is valuable. Some thoughts on it:
1. Backticks are carried over through decades and used to run commands in many shells and even general-purpose languages: Perl, Ruby
2. Even in bash, where they are used the most, it's common to move away from backticks to `$(...)`
3. In nu backticks are limited to the single-word commands: you can do `` `date` ``, but you cannot do `` `touch foo` ``.
4. You can achieve the same with `run-external` (with the same limitations). Backticks seem to be just an alias for `run-external`
5. You can call an external command with `^` as well, combining both internal and external commands: `touch (^date +%Y%m%d)`

So what if backticks were removed as a command runner and kept as string delimiters instead?

**Comment by amtoine** [comment]
to the best of my knowledge, path are a nightmare on Windows and backticks are the current solution to solve this :thinking: 

there have been ongoing discussions about the quotes, double quotes and backticks and how they related to strings, extrapolation and paths :+1: 

as for you question, i personnally never use backticks to run commands in Nushell :open_mouth:

**Comment by fdncred** [comment]
Backticks help with paths with spaces in them across platforms. I'm not sure if that's what you mean by "string delimiter" or not. I friggin hate backticks. My raw string PR probably would've been better.

**Reply by hustcer** [reply]
Maybe we can keep both?

**Reply by ineu** [reply]
> I'm not sure if that's what you mean by "string delimiter" or not. 

It is, as described here: https://www.nushell.sh/book/working_with_strings.html#string-formats-at-a-glance
